CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## ✨ Features
 * Add ability to insert images into text labels using a format expression: `"text-field": ["format", "Some text", ["image", "my-image"], "some more text"]` ([#8904](https://github.com/mapbox/mapbox-gl-js/pull/8904))
```

Comment:
using a `format` expression?

Reply:
would "using an `image` expression within a `format` expression" be better?

Reply:
yep!

Reply:
Or were you talking about type of quotes?

Reply:
oh I was talking about the formatting. but I like your suggestion